### ISSUE TITLE
Fine tune model based on TimeGPT

### ISSUE THREAD

**Issue by Imortalco** [Imortalco/AITradingBot#2]
Read about : 

- [ ] - Exogenous variables
- [x] - finetune_steps param in forecast method
- [x] - Fine-tuning with a specific loss function
- [ ] - Anomaly detection

**Comment by Imortalco** [comment]
Added fine tune stapes and specific loss function ('mae'). Next step is probably to add hyperparameter tuning. IDK
